ha/cli/commands.py

```diff
@@ -58,32 +58,32 @@ def add_args(component_parser):
             component_parser (<class 'argparse._SubParsersAction'>): argparse to add subparser.
         """
         cluster_parser =  component_parser.add_parser("cluster",
-            help = "Manage cluster")
+            help = "Manage cluster Operation like start, stop cluster.")
         action_subparser = cluster_parser.add_subparsers(
             title="Cluster Actions",
             dest="cluster_action")
 
         add_node_parser = action_subparser.add_parser("add_node",
-            help = "Add node to cluster")
+            help = "Add new node to existing cluster to manage their service.")
         add_node_parser.add_argument("node",
-            help="Node name", action="store")
+            help="Provide node name to add it in cluster.", action="store")
 
         remove_node_parser = action_subparser.add_parser("remove_node",
-            help = "Remove node to cluster")
+            help = "Remove node from cluster")
         remove_node_parser.add_argument("node",
-            help="Node name", action="store")
+            help="Provide node name to remove it in cluster.", action="store")
 
         start_parser = action_subparser.add_parser("start",
-            help = "Start cluster")
+            help = "Start cluster and it's serives.")
```

Comment:
serives -> services

Reply:
Done

Reply:
I do not see this fixed yet @ajay-paratmandali ?

Reply:
Done